src/__tests__/useSubscription-test.tsx

```diff
@@ -0,0 +1,349 @@
+import { ApolloClient } from 'apollo-client';
```

Comment:
It would be nice to merge master into this PR. After that the following changes will be required:

1. `flushEffects` won't be required any more and it will be possible to remove all invocations of it.
2. `flushEffectsAndWait` should be replaced with `wait` from `../__testutils__/wait`.
3. You could add `jest.mock('../internal/actHack');` just below the imports to apply the _act hack_ (please look at the review of _useSubscription-test.tsx_ for more details).

After applying those changes it could look like that file: https://gist.github.com/trojanowski/9bb29c77537371919fd861cd47cb5015#file-usesubscription-test-tsx